tmt/utils.py

```diff
@@ -518,6 +518,17 @@ def workdir(self):
             create_directory(self._workdir, 'workdir', quiet=True)
         return self._workdir
 
+    def load_run(self, run):
+        try:
+            run.load_from_workdir()
+        except GeneralError as error:
+            self.warn(f'Failed to check {run.workdir} ({error}).')
+            return False
+        for plan in run.plans:
+            for step in plan.steps(disabled=True):
+                step.load()
+        return True
+
```

Comment:
Defining the `load_run()` method in the `Common` class means that every object in the [hierarchy](https://tmt.readthedocs.io/en/latest/contribute.html#class-structure) will inherit it, although it is relevant for `Status` and `Clean` only. Could we find a better place for it?

Reply:
I will transform it into a function in `utils.py` rather than a method. It won't log inside the function but `Status` and `Clean` can take care of the warning, that won't be a big deal (only one logging command duplicated)